templates/telco/osm/onboarding-network-functions.html

```diff
@@ -0,0 +1,191 @@
+      {{ image (
+        url="https://assets.ubuntu.com/v1/78a94857-Onboarding+1.svg",
+        alt="",
+        width="350",
+        height="280",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <h2>3 easy steps to onboard your network function</h2>
+		<p>Onboarding is all about configuring the day 0, day 1 and day 2 operations. All the life cycle stages can be achieved by defining these operations properly in the OSM packages for the network functions. The <a href="https://osm.etsi.org/docs/vnf-onboarding-guidelines/" class="p-link--external">process</a> includes:</p>
+		<hr  />
+		<div class="row">
+			<div class="col-6">
+				<h2>Day 0</h2>
+			</div>
+			<div class="col-6">
+				<ul>
+					<li>Create the network function descriptors as templates to instantiate/terminate the service.</li>
+					<li>Create packages for network services i.e. templates for images and configurations of the virtualized infrastructure using Enhanced Platform Awareness provided by OSM.</li>
+					<li>Integrate the  cloud-init scripts in the descriptors for primary configurations like OS boot, SSH and user management.</li>
+			  </ul>
+			</div>
+		</div>
+		<hr  />
+		<div class="row">
+			<div class="col-6">
+				<h2>Day 1</h2>
+			</div>
+			<div class="col-6">
+				<ul>
+					<li>Create a <a href="https://juju.is/docs/sdk/hello-world" class="p-link--external">charm</a> (proxy or native) as an operator for the network function.</li>
+					<li>Define the primitives/actions in charms that need to be performed right after the instantiation of service.</li>
+					<li>Integrate the configured charm in the OSM descriptors.</li>
+			  </ul>
+			</div>
+		</div>
+		<hr  />
+		<div class="row">
+			<div class="col-6">
+				<h2>Day 2</h2>
+			</div>
+			<div class="col-6">
+				<ul>
+					<li>Reconfigure when needed after the service is running</li>
+					<li>Monitor the specific metrics for the infrastructure</li>
+					<li>Scale on the basis of monitoring analysis</li>
+					<li>Operate to enable closed-loop automation</li>
+				</ul>
+			</div>
+		</div>
+    <div class="row">
+      <div class="col-12 u-vertically-center u-hide--small">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/e8f2394f-Onboarding+2.svg",
+          alt="",
+          width="1000",
+          height="180",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
+      </div>
+    </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-vertically-center">
+    <div class="col-12">
+      <h2>Lifecycle management</h2>
+      <p>The goal of onboarding is to create a package which can fulfil the stages to manage the lifecycle of network services. The package should  include network function instantiation, scaling in/out, healing or recovery, updating, operating and termination.</p>
+    </div>
+		<hr  />
```

Comment:
We have [the `p-separator`](https://vanillaframework.io/docs/base/separators#separator) for this.

Reply:
`p-separator` is giving too much space compared to [the design](https://github.com/canonical-web-and-design/web-squad/issues/4153)?

Reply:
@anasereijo Using `p-separator` seems to do the job of adding more space. Are you ok with this?